test/integration/content-explorer/ContentExplorer.int.test.js

```diff
@@ -192,14 +192,14 @@ describe('ContentExplorer', () => {
             helpers.checkRowSelections(1);
             helpers.selectRow(4);
             helpers.checkRowSelections(4);
-            helpers.previewItemFromRow(2);
```

Comment:
This is a rather large 3d file that can occasionally time out. I just switched the test to use smaller files.

Reply:
do you think this could be why we were occasionally seeing this error? https://docs.cypress.io/guides/references/error-messages.html#The-Chromium-Renderer-process-just-crashed

Reply:
No idea. I guess we'll find out!